src/overlays/gamestates/ovl_file_choose/z_file_choose_NES.c

```diff
@@ -4,96 +4,2362 @@
+            }
+        } else {
+            j = D_80814650[this->confirmButtonIndex];
+        }
+
+        this->windowContentVtx[i + 0].v.ob[0] = this->windowContentVtx[i + 2].v.ob[0] = this->windowPosX - 0xA;
+
+        this->windowContentVtx[i + 1].v.ob[0] = this->windowContentVtx[i + 3].v.ob[0] =
+            this->windowContentVtx[i + 0].v.ob[0] + 0x48;
+
+        this->windowContentVtx[i + 0].v.ob[1] = this->windowContentVtx[i + 1].v.ob[1] =
+            this->windowContentVtx[j].v.ob[1] + 4;
+
+        this->windowContentVtx[i + 2].v.ob[1] = this->windowContentVtx[i + 3].v.ob[1] =
+            this->windowContentVtx[i + 0].v.ob[1] - 0x18;
+
+        this->windowContentVtx[i + 1].v.tc[0] = this->windowContentVtx[i + 3].v.tc[0] = 0x900;
+
+        this->windowContentVtx[i + 2].v.tc[1] = this->windowContentVtx[i + 3].v.tc[1] = 0x300;
+    }
+
+    this->windowContentVtx[i + 4].v.ob[0] = this->windowContentVtx[i + 6].v.ob[0] = this->windowPosX + 0x3A;
+
+    this->windowContentVtx[i + 5].v.ob[0] = this->windowContentVtx[i + 7].v.ob[0] =
+        this->windowContentVtx[i + 4].v.ob[0] + 0x80;
+
+    this->windowContentVtx[i + 4].v.ob[1] = this->windowContentVtx[i + 5].v.ob[1] =
+        this->windowContentVtx[D_80814638[this->warningButtonIndex]].v.ob[1];
+
+    this->windowContentVtx[i + 6].v.ob[1] = this->windowContentVtx[i + 7].v.ob[1] =
+        this->windowContentVtx[i + 4].v.ob[1] - 0x10;
+
+    this->windowContentVtx[i + 5].v.tc[0] = this->windowContentVtx[i + 7].v.tc[0] = 0x1000;
+}
+#else
+void FileSelect_SetWindowContentVtx(GameState* thisx);
+#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_file_choose/FileSelect_SetWindowContentVtx.s")
+#endif
+
+u16 D_80814654[] = {
+    0x88,
+    0x194,
+    0x2A0,
+};
+TexturePtr sFileSelRemainsTextures[] = {
+    gFileSelOdolwasRemainsTex,
+    gFileSelGohtsRemainsTex,
+    gFileSelGyorgsRemainsTex,
+    gFileSelTwinmoldsRemainsTex,
+};
+TexturePtr sFileSelDayENGTextures[] = {
+    gFileSelFirstDayENGTex,
+    gFileSelFirstDayENGTex,
+    gFileSelSecondDayENGTex,
+    gFileSelFinalDayENGTex,
+};
+TexturePtr sFileSelHeartPieceTextures[] = {
+    gFileSel0QuarterHeartENGTex,
+    gFileSel1QuarterHeartENGTex,
+    gFileSel2QuarterHeartENGTex,
+    gFileSel3QuarterHeartENGTex,
+};
+static TexturePtr sHeartTextures[2][5] = {
+    {
+        gHeartEmptyTex,
+        gHeartQuarterTex,
+        gHeartHalfTex,
+        gHeartThreeQuarterTex,
+        gHeartFullTex,
+    },
+    {
+        gDefenseHeartEmptyTex,
+        gDefenseHeartQuarterTex,
+        gDefenseHeartHalfTex,
+        gDefenseHeartThreeQuarterTex,
+        gDefenseHeartFullTex,
+    },
+};
+u8 sHealthToQuarterHeartCount[] = {
+    0, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3,
+};
+s16 sFileSelRupeePrimColors[3][3] = {
+    { 200, 255, 100 },
+    { 170, 170, 255 },
+    { 255, 105, 105 },
+};
+s16 sFileSelRupeeEnvColors[3][3] = {
+    { 0, 80, 0 },
+    { 10, 10, 80 },
+    { 40, 10, 0 },
+};
```

Comment:
I assume this is a wallet thing, similar question to the half damage hearts one above. If so, could do with attaching comments for normal/adult/giant's.